docs/mobile-apps/features/audio-capture.md

```diff
@@ -0,0 +1,81 @@
+---
+id: audio-capture
+title: Audio Capture
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
```

Comment:
@aoleszyn Please add **Real Device only** badge. You can find an example [here](https://docs.saucelabs.com/mobile-apps/features/camera-image-injection/).

Reply:
Badge added.